City-of-Helsinki/drupal-helfi-rekry
# UHF-10581

Added test features

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10581`
  * `make fresh`

## How to test

- Create new job listing, set  publishing time from sidebar and save as unpublished
- After the scheduled time, run drush cron
- Run drush sqlc and `select * from queue`, you should see the indexing job in the table
